### ISSUE TITLE
Breaking in a loop inside a kernel

### ISSUE THREAD

**Issue by jc211** [NVIDIA/warp#121]
Adding a break inside a for loop in the kernel raises an error. 

Example:
```python
import warp as wp
@wp.kernel
def foo(a: int):
    for i in range(10):
        wp.printf("a: %d, i: %d\n", a, i)
        if i == a:
            break

wp.init()
wp.launch(kernel=foo, dim=(1,), inputs=[3])
wp.synchronize_device()
```

**Comment by jc211** [comment]
Getting around this would probably require something like: ```while(i != a)``` instead of a for loop

**Comment by mmacklin** [comment]
Hi @jc211, this has been addressed in b57868fac70cce0e585ee0053f038fc3c9e25356.

Thanks,
Miles
